model-server/src/main/kotlin/org/modelix/model/server/handlers/ModelReplicationServer.kt

```diff
@@ -147,6 +148,37 @@ class ModelReplicationServer(
             call.respondDelta(versionHash, baseVersionHash)
         }
 
+        delete<Paths.deleteRepositoryBranch> {
+            val repositoryName = call.parameters["repository"]
+            if (repositoryName == null) {
+                call.respondText("Request lacks repository name", status = HttpStatusCode.BadRequest)
+                return@delete
```

Comment:
thought: It is unfortunate that we can not just throw the `io.ktor.server.plugins.BadRequestException`.

We would need to refactor the usage of the StatusPage plugin
See https://github.com/modelix/modelix.core/blob/main/authorization/src/main/kotlin/org/modelix/authorization/KtorAuthUtils.kt#L88

Reply:
Yes, the lack of a default HTTP exception hierarchy in ktor is a pain. However, we should probably split that refactoring (for the whole implementation) into a separate PR?

Reply:
Definitly. We will propably also implement https://issues.modelix.org/articles/MODELIX-A-83/0009-Use-application-problemjson-for-error-responses-in-REST-APIs when cleaning it up.